src/coreclr/jit/unwind.cpp

```diff
@@ -311,7 +311,6 @@ void Compiler::unwindEmitFuncCFI(FuncInfoDsc* func, void* pHotCode, void* pColdC
 #endif // DEBUG
 
     assert(endOffset <= info.compTotalHotCodeSize);
-
```

Comment:
Looks like the only change in unwind.cpp is an arbitrary whitespace change. Maybe remove unwind.cpp from the PR.